### ISSUE TITLE
[IDE Updater] IDE unable to find new updates

### ISSUE THREAD

**Issue by tabularelf** [YoYoGames/GameMaker-Bugs#8673]
### Description
I have previously noticed this at least 2 beta versions ago, but as I was aware there was several of issues with the IDE Updater, I had briefly waited to see if .710 would remedy this. Comes .710 and .713, and despite installing .710, I'm unable to get any updates whatsoever. I have waited days, weeks even in the last beta versions, just to wait out DNS.

### Steps To Reproduce
1. Load up GameMaker	 (at least one version before the latest)
2. Goto Help -> Check for Updates
3. See "No New Versions Were Found. The IDE is Up To Date."

### Which version of GameMaker are you reporting this issue for?
IDE v2024.1100.0.710 Runtime v2024.1100.0.719

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

### Attached Files
* GameMaker-Beta_2024-12-12_13-15-02.png

[4856fd1f-5970-4672-830b-fbf4c8d2a97d](https://api.gamemaker.io/api/github/downloads/4856fd1f-5970-4672-830b-fbf4c8d2a97d)

**Comment by stuckie** [comment]
From the ui.log, it looks like you've overridden the rss feed it's using to check for IDE versions.
May want to double check it's pointing towards the real one, and not a fake one.
If you still have issues, please submit another set of logs, with it trying the correct rss feed!

**Comment by tabularelf** [comment]
> From the ui.log, it looks like you've overridden the rss feed it's using to check for IDE versions. May want to double check it's pointing towards the real one, and not a fake one. If you still have issues, please submit another set of logs, with it trying the correct rss feed!

Oh right, I forgot about that. Those RSS feeds are definitely legit, but I had them assigned from a period of them when I was using a separate code editor. In any case, removing those RSS feeds hasn't improved the situation.

https://api.gamemaker.io/api/github/downloads/f672957f-2cfd-4870-b518-42206ae2566d

**Comment by tabularelf** [comment]
Upon further digging (as no response was given), I've found out that `default_macros.json` from %ProgramDatas% was modified. But since this wasn't of my own ill modification in the past, I suspect it is involving a particular editor I have used.

Whatever the case may be, this isn't a GameMaker specific issue. Though I'm leaving this as a note, just in case anyone else happens to run into it.

**Comment by tabularelf** [comment]
I can confirm, it's Stitch from VSCode that is deliberately adding it and not me. It at least solves one mystery.
It's noted in the source code to "Prevent the IDE from showing an update prompt on boot", but I feel that there should be a much better approach for this.

**Comment by stuckie** [comment]
Ah perfect.. glad you figured out what it was, in the end!
For reference, the real rss is the same, just without the obviously fake part tagged on to it.

As for the update notifications, you can ignore them and tell it not to update for that version, and with the new update process, you can now manually check for updates even if you have skipped it, so it is a bit better than previous versions. Hopefully that works out better, rather than the slightly naughty fiddling with the .rss feed to block it!

**Comment by tabularelf** [comment]
Yeah that was pretty obvious when I had probed the RSS feed myself, minus `TOTALLY-FAKE`. It's honestly a lil bit bothersome since Stitch does this to deliberately avoid IDE prompts. I've informed the creators anyway about it, and given my 2 cents on the thoughts. It's not super hard for me to workaround the issue (as it's just not clicking the "Open GameMaker project" from VSCode), but since other people do use Stitch with VSCode, this could be causing interference with them too.
